obs-packaging/runtime/update.sh

```diff
@@ -61,19 +61,22 @@ info "ksm-throttler ${KSM_THROTTLER_REQUIRED_VERSION}"
 KATA_IMAGE_REQUIRED_VERSION=$(pkg_version "${kata_osbuilder_version}" "" "")
 info "image ${KATA_IMAGE_REQUIRED_VERSION}"
 
-KATA_QEMU_LITE_REQUIRED_VERSION=$(pkg_version "${qemu_lite_version}" "" "${qemu_lite_hash}")
-info "qemu-lite ${KATA_QEMU_LITE_REQUIRED_VERSION}"
-
 KATA_QEMU_VANILLA_REQUIRED_VERSION=$(pkg_version "${qemu_vanilla_version}" "" "${qemu_vanilla_hash}")
 info "qemu-vanilla ${KATA_QEMU_VANILLA_REQUIRED_VERSION}"
 
+if [ "$arch" == "x86_64" ]; then
+	KATA_QEMU_LITE_REQUIRED_VERSION=$(pkg_version "${qemu_lite_version}" "" "${qemu_lite_hash}")
+	info "qemu-lite ${KATA_QEMU_LITE_REQUIRED_VERSION}"
+	replace_list=("qemu_lite_version=${KATA_QEMU_LITE_REQUIRED_VERSION}")
```

Comment:
Nit: It would be better to define the array outside the `if`:

```sh
replace_list=()

if [ "$arch" == "x86_64" ]; then
    replace_list+=( ... )
fi

replace_list+=( ... )
```